parse/DoubleComplexValueRefParser.cpp

```diff
@@ -48,7 +48,7 @@ namespace parse {
 
             species_empire_opinion
                 = (
-                    (  tok.SpeciesOpinion_ [ _a = construct<std::string>("SpeciesEmpireOpinion") ]
```

Comment:
@adrianbroher Since this affects building with boost 1.63, could you expand on a preferred resolution? (or `-DBOOST_PHOENIX_NO_VARIADIC_EXPRESSION`)